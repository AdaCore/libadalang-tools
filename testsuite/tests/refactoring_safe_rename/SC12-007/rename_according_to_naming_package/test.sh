#!/bin/sh
safe_rename  -P test.gpr -S Test.2.ada -L 1 -R 6 -N Fox --algorithm analyse_ast
