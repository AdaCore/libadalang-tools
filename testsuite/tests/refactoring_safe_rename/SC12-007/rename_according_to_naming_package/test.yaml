description: |
    This test checks if the safe rename tool correctly handles
    use the Naming package from project file.

driver: shell_script
