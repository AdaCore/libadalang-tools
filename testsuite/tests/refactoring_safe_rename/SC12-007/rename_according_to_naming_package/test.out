# Renaming node Test.2.ada:1:6: 
References:
Unit:Foo--Bar--Baz.1.ada
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 9
End_Line: 1
End_Column: 12
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 5
End_Line: 2
End_Column: 8
Unit:Foo--Bar.1.ada
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 9
End_Line: 1
End_Column: 12
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 5
End_Line: 2
End_Column: 8
Unit:Foo.1.ada
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 9
End_Line: 1
End_Column: 12
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 5
End_Line: 2
End_Column: 8
Unit:Test.2.ada
Line: 1
Source Location Range:
Start_Line: 1
Start_Column: 6
End_Line: 1
End_Column: 9
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 6
End_Line: 2
End_Column: 9
Line: 3
Source Location Range:
Start_Line: 3
Start_Column: 6
End_Line: 3
End_Column: 9
File renames:
Foo--Bar--Baz.1.ada -> Fox--Bar--Baz.1.ada
Foo--Bar.1.ada -> Fox--Bar.1.ada
Foo.1.ada -> Fox.1.ada

